batch/batch/worker.py

```diff
@@ -861,7 +861,7 @@ async def run(self):
             await site.start()
 
             try:
-                await asyncio.wait_for(self.activate(), MAX_IDLE_TIME_MSECS * 1000)
+                await asyncio.wait_for(self.activate(), MAX_IDLE_TIME_MSECS)
```

Comment:
Either I'm misreading the docs for `wait_for` or MAX_IDEL_TIME_MSECS is not in milliseconds, which is it?

Reply:
I was being stupid. We need to divide by 1000.